### ISSUE TITLE
Not able to get data indexed in lucene

### ISSUE THREAD

**Issue by priti-patil** [rnewson/couchdb-lucene#261]
I am trying to use couchdb-lucene. I followed steps mentioned in documentation. My coucdb-lucene is working fine if as I can see message when I open localhost:5985. Additionally I configured global handlers in local.ini as below:
[httpd_global_handlers]
_fti = {couch_httpd_proxy, handle_proxy_req, <<"http://127.0.0.1:5985">>}

I created new view by adding index everything code in its definition. However I am not able to figure out if data is getting indexed in lucene or not and how can I query it. Note that I am using couchdb 2.1.1 version.

**Comment by ryanMeredith** [comment]
From what I have read from the previous comments on here it is not possible to use the proxy endpoint on 5984 although reading the couch docs suggests you can.

http://docs.couchdb.org/en/2.1.1/externals.html

going directly to Lucene works though. It is a pain to change the endpoint in the code though.
http://127.0.0.1:5985/local/**dbname**/_design/foo/by_subject?q=hello

If anyone knows any different please say :)

**Comment by rnewson** [comment]
it's true that couchdb has dropped those kinds of handlers since 2.x.

You could use nginx or haproxy to provide a single namespace if you like, or you could look at my couchdb 2.x / lucene integration instead of couchdb-lucene (https://github.com/cloudant-labs/dreyfus / https://github.com/cloudant-labs/clouseau).

**Comment by ryanMeredith** [comment]
Thanks for your work on these.
I have to confess I tried following the instructions here about a year ago and failed miserably.  https://developer.ibm.com/dwblog/2015/text-search-apache-couchdb/

I will give it another go tomorrow, maybe I am older and wiser now.

**Comment by rnewson** [comment]
Yeah, it's tricky. clouseau/dreyfus was originally an internal Cloudant feature and I had the latitude to embed it pretty tightly.

**Comment by stickbyatlas** [comment]
What I really like about this plugin, vs the clouseau/dreyfux approach, is that this plugin works without having to build your CouchDB from scratch. I enjoyed being able to download an installer package for OS X from the CouchDB website, running the instructions from the README, modifying a few config settings, and being able to query away. For me, this is totally worth pushing CouchDB behind a proxy in order to maintain a single namespace.

I wish the clouseau/dreyfus approach was as straightforward. In the meantime, stubborn me is going to keep implementing full text search using this repo. @rnewson Thanks for this project.
